src/main/kotlin/no/nav/familie/ef/iverksett/domene/AndelTilkjentYtelse.kt

```diff
@@ -0,0 +1,53 @@
+package no.nav.familie.ef.iverksett.domene
+
+import no.nav.familie.ef.iverksett.økonomi.KjedeId
+import no.nav.familie.ef.iverksett.økonomi.PeriodeId
+import no.nav.familie.ef.iverksett.økonomi.tilNullAndelTilkjentYtelse
+import java.time.LocalDate
+
+data class AndelTilkjentYtelse(val beløp: Int,
+                               val stønadFom: LocalDate, /// TODO  Gjør nullable
+                               val stønadTom: LocalDate, /// TODO  Gjør nullable
+                               val personIdent: String,
+                               val periodeId: Long? = null,
+                               val forrigePeriodeId: Long? = null,
+                               ) {
```

Comment:
Trenger vi `val kildeBehandlingId: UUID? = null` her også @blommish ?

Reply:
Tror ikke det nei :) 
Alternativet her er att vi faktiskt lar iverksett håndtere allt med økonomi? Slik att vi kun lagrer tilkjent ytelse og andel her? Usikker

Reply:
Jeg tenker at vi er nødt til å la ef-sak kjøre konsistensavstemming, men vi kan lage best mulig api for det i ef-iverksett. Poenget er jo at man får sjekket at de saksbehandlingssystemet er enig med økonomisystemet om hvilke utbetalinger som gjelder. 

Men jeg håper at grensesnittavstemmingen kan skje fra ef-iverksett, i hvert fall